src/pages/Facility/settings/healthcareService/HealthcareServiceShow.tsx

```diff
@@ -40,6 +51,23 @@ export default function HealthcareServiceShow({
         healthcareServiceId,
       },
     }),
+    enabled: !isDeleted,
```

Comment:
why disable the query when deleted? 🤔 we should ideally navigate away from this component instead right?